### ISSUE TITLE
Adding tests for clean_data function issue

### ISSUE THREAD

**Issue by nikizamani** [DSCI-310/DSCI-310-Group-2-packages#9]
I edited the test-clean_data.R accidentally on main and not on the branch I created for it. It didn't ask for any approval. It should be okay right?

**Comment by dliviya** [comment]
let me just double check, do you know what you edited?

**Comment by dliviya** [comment]
ok I looked at the changes, it should be ok, I do not think i did the right protection settings for this repo

**Comment by nikizamani** [comment]
yeah, I added tests for the test-clean_data.r

**Comment by nikizamani** [comment]
yeah, I was wondering that because it didn't give me a warning before pushing

**Comment by dliviya** [comment]
I just updated the protection settings, I don't think it will let you do that again. Thanks for pointing it out!

**Comment by nikizamani** [comment]
awesome! thanks
